src/Connector.sol

```diff
@@ -89,17 +89,20 @@ contract CentrifugeConnector {
     }
 
     // --- Outgoing message handling ---
-    function transferToCentrifuge(uint64 poolId, bytes16 trancheId, bytes32 destinationAddress, uint128 amount)
-        public
-    {
+    function transferTrancheTokensToCentrifuge(
+        uint64 poolId,
+        bytes16 trancheId,
+        bytes32 destinationAddress,
+        uint128 amount
+    ) public {
         RestrictedTokenLike token = RestrictedTokenLike(tranches[poolId][trancheId].token);
         require(address(token) != address(0), "CentrifugeConnector/unknown-token");
 
         require(token.balanceOf(msg.sender) >= amount, "CentrifugeConnector/insufficient-balance");
         token.burn(msg.sender, amount);
 
         router.send(
-            ConnectorMessages.formatTransfer(
+            ConnectorMessages.formatTransferTrancheTokens(
```

Comment:
Changing this name since in v2 we got 2 transfer messages; `Transfer` for stable-coin transfers and `TrancheTrancheTokens`

Reply:
@AStox have a look at this part of the spec: https://centrifuge.hackmd.io/Soy4JiFbRSyovj53KW-a4A#Transfer

Reply:
Got it! Thanks